src/models/Project.ts

```diff
@@ -272,10 +273,13 @@ class Project {
     };
 
     validateMER(): ValidationError {
+        const selected = this.dataElementsSelection.getAllSelected();
+
         return _.concat(
-            this.dataElementsMER.validateAtLeastOneSelected(this.sectors),
-            this.dataElementsMER.validateMaxSelectedBySector(this.sectors, 3),
-            this.dataElementsMER.validateMaxSectorsWithSelections(this.sectors, 3)
+            this.dataElementsMER.validateTotalSelectedCount(this.sectors, {
+                min: isTest() ? 1 : Math.min(selected.length, 3),
```

Comment:
@tokland Do you want to keep this 'isTest'? I am concerned we will forget about it at a certain point in time.

Reply:
I am aware. The validation changes broke the fixtures projects we had and I didn't want to spend too much time at that point, thus the `isTest`. I'll take a note, if there is time I'll refactor it.